spring-resteasy/pom.xml

```diff
@@ -153,7 +153,7 @@
         <dependency>
             <groupId>jakarta.ws.rs</groupId>
             <artifactId>jakarta.ws.rs-api</artifactId>
-            <scope>provided</scope>
+<!--            <scope>provided</scope>-->
```

Comment:
why the change of scope?

Reply:
@emmartins I have met several testing problems locally so I'm modifying something to make the example works, so I moved the PR into draft status. Because the final version of the draft may change a lot.

Reply:
The provided scope should enable the dependency on compile and testing phases, only removes it from runtime/packaging, which is how it should be done. Please retry with provided scope, maybe you did more changes after that fixed it?